models/org.go

```diff
@@ -495,7 +499,7 @@ func (org *User) GetUserRepositories(userID int64, page, pageSize int) ([]*Repos
 	repos := make([]*Repository, 0, pageSize)
 	// FIXME: use XORM chain operations instead of raw SQL.
 	if err = x.Sql(fmt.Sprintf(`SELECT repository.* FROM repository
```

Comment:
I prefer 1 PR = 1 problem solved.

I will replace raw queries  with XORM chain operations in another PR.